app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
+  def execute(query)
```

Comment:
Yeah, you're totally spot on. Was trying to figure out how to do this right, but I can't think of any sort of wall we could put up that would actually make sure nothing personal got out. SQL is Turing-complete, so I imagine that doing this properly would be just as difficult as sandboxing any other sort of program.

I think our best bet for now is to just keep it laden with warnings and leave it to the user to not abuse this. The only folks that'll have access will be staff, after all. And all of the activity is mirrored to the mirror channel.